### PR TITLE
Add annotation support

### PR DESCRIPTION
**Pull request by Loori-R**
- #30

### CODE REVIEW

File: src/datasource.tsx [hunk 1]
```diff
@@ -83,7 +82,7 @@ import { PrometheusVariableSupport } from './variables';
 enum PromApplication {
   VictoriaMetrics = 'VictoriaMetrics',
 }
-const ANNOTATION_QUERY_STEP_DEFAULT = '60s';
```
Comment (dmitryk-dk) [inline src/datasource.tsx:86]: I don't think that is good idea export const from main functionality. Maybe it is better to move that const to consts.ts file and export from there?

**Review by dmitryk-dk — APPROVED** [review]
LGTM! See small comment

File: src/datasource.tsx (hunk 1 above)
Comment (Loori-R) [reply src/datasource.tsx:86]: Good idea! Updated

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Comment by Loori-R** [comment]
@hagen1778 Sure! I'll do it.

**Pull request merged**